Acd 600 (#15)
Actindo-AG/Documentation: Taxes/UserInterface/01b_DecisionMatrix.md

# Decision matrix

*Taxes > Settings > Tab DECISION MATRIX*

![Decision matrix](../../Assets/Screenshots/Taxes/Settings/DecisionMatrix/DecisionMatrix.png "[Decision matrix]")

- ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add)   
Click this button to create an entry. A new input line is displayed.

- ![Drag](../../Assets/Icons/Points03.png "[Drag]") (Drag)  
Click this button to move the entry to a different position with the left mouse button pressed. Drop it in the desired position by releasing the mouse button.

- *VAT ID*  
Value-added tax identification number. In this case, the VAT ID refers to the buyer's VAT ID number. Click the drop-down list to select the appropriate option. The following options are available:

  - **No**  

>   Select this option if the end customer is a private individual and, therefore, has no VAT ID number (business-to-customer transaction).

Author: ```suggestion
  Select this option if the customer is a private individual and, therefore, has no VAT ID number (B2C transaction).
```

  - **Yes**  

>   Select this option if the end customer is a company and, therefore, has a VAT ID number (business-to-business transaction).

Author: ```suggestion
  Select this option if the customer is a company and, therefore, has a VAT ID number (B2B transaction).
```

# Decision matrix

*Taxes > Settings > Tab DECISION MATRIX*

![Decision matrix](../../Assets/Screenshots/Taxes/Settings/DecisionMatrix/DecisionMatrix.png "[Decision matrix]")

- ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add)   
Click this button to create an entry. A new entry line is displayed.

- ![Drag](../../Assets/Icons/Points03.png "[Drag]") (Drag)  
Click this button to move the entry to a different position with the left mouse button pressed. Drop it in the desired position by releasing the mouse button.

- *VAT ID*  
Value-added tax identification number. In this case, the VAT ID refers to the buyer's VAT ID number. Click the drop-down list to select the appropriate option. The following options are available:

>   - **Yes**  
>   Select this option if the end customer is a company and, therefore, has a VAT ID number (business-to-business transaction).
>   - **No**  
>   Select this option if the end customer is a private individual and, therefore, has no VAT ID number (business-to-customer transaction).

Reviewer: ```suggestion
  transaction).
  - **No**  
  Select this option if the end customer is a private individual and, therefore, has no VAT ID number (business-to-customer transaction).
  - **Yes**  
  Select this option if the end customer is a company and, therefore, has a VAT ID number (business-to-business 
```

- *Country of origin*  
Country where the product is shipped from.

  - Click the drop down-list on the left to select a group of countries. As a group of countries, only the **EU - European Union** option is available. When selecting this option, the drop-down list on the right is locked. Leave it blank (**Any**) if you want to specify an individual country in the drop-down list on the right.

  - Click the drop-down list on the right to select an individual country. All available countries in the system are displayed.

- *Ship-to country*  
Country where the product is shipped to.

  - Click the drop down-list on the left to select a group of countries. As a group of countries, only the **EU - European Union** option is available. When selecting this option, the drop-down list on the right is locked. Leave it blank (**Any**) if you want to specify an individual country in the drop-down list on the right.

  - Click the drop-down list on the right to select an individual country. All available countries in the system are displayed.

- *Ship-to postcode*  
Specific area within a country or territory where the product is shipped to.

>   - Click the drop down-list on the left and select the appropriate option. The following options are available:

Reviewer: ```suggestion
  - Click the drop-down list and select the appropriate option. The following options are available:
```

    - **Range**  
    Select this option to specify a numeric postcode range, for example 27498 or 52001-52006.
    - **Regex**  
    Select this option to specify a postcode containing alphanumeric characters with regular expressions, for example /^BT.*/.   

> 

Reviewer: ```suggestion
The field to the right of the *Ship-to postcode* drop-down list is unlocked.
```

>   - Enter the applicable range or regular expression in the field on the right.

Reviewer: ```suggestion
  - Enter the applicable range or regular expression in the unlocked field on the right.
```

- *Net value of goods smaller than*  

> Relevant top amount for a criterion to be applied.    
> 
>   - Click the drop-down list on the left to select the applicable currency.   
> 
>   - Enter a maximum net value amount using the keyboard or the increase and decrease arrows on the right.

Reviewer: ```suggestion
Relevant top amount for a criterion to be applied.  Click the drop-down list to select the applicable currency. All available currencies are displayed in the list.  When selecting a currency, the field to the right of the *Net value of goods smaller than* drop-down list is unlocked. Enter a maximum net value amount in the field by using the keyboard or by clicking the increase and decrease arrows on the right of the field.
```

> # Decision matrix

Reviewer: Schaue ich nach deinem Gespräch mit Flo durch
